packages/scandipwa/src/component/WishlistItem/WishlistItem.container.js

```diff
@@ -158,6 +159,41 @@ export class WishlistItemContainer extends PureComponent {
             );
     }
 
+    recalculatePrice() {
+        const {
+            product: {
+                type_id,
+                items
+            }
+        } = this.props;
+
+        if (type_id !== GROUPED || !items) {
+            return;
+        }
+
+        const { product: { price_range: { minimum_price } } } = this.props;
+
+        items.forEach((link) => {
+            const {
+                qty, product: {
+                    price_range: {
+                        minimum_price: link_minimum_price
+                    }
+                }
+            } = link;
+
+            // eslint-disable-next-line fp/no-let
+            let firstRun = true;
```

Comment:
No-let is a strict policy. Please remove this.

Reply:
Removed